plugins/modules/rma_workflow_manager.py

```diff
@@ -42,6 +42,7 @@
   - cisco.dnac.workflow_manager_params
 author:
   - Trupti A Shetty (@TruptiAShetty)
+  - Ajith Andrew J (@ajithandrewj)
```

Comment:
Please append the author names instead of inserting them.

Reply:
Updated correctly